<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parameter</class>
 <widget class="QDialog" name="Parameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>parameter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>name</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="custom_name">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>custom</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="custom_name_value"/>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="sampler_name">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>sampler</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="sampler_name_value">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="auto_name">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>auto</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="auto_name_value">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="common_name">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>common</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="common_name_value">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="attribute_name">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>attribute</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="attribute_name_value">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="type_value"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>size</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="size_value"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>scale</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="scale_value">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1024</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>qualifier</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="qualifier_value"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>custom_name_value</tabstop>
  <tabstop>sampler_name_value</tabstop>
  <tabstop>auto_name_value</tabstop>
  <tabstop>common_name_value</tabstop>
  <tabstop>attribute_name_value</tabstop>
  <tabstop>type_value</tabstop>
  <tabstop>qualifier_value</tabstop>
  <tabstop>size_value</tabstop>
  <tabstop>scale_value</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>Parameter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>Parameter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_name</sender>
   <signal>toggled(bool)</signal>
   <receiver>custom_name_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sampler_name</sender>
   <signal>toggled(bool)</signal>
   <receiver>sampler_name_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>auto_name</sender>
   <signal>toggled(bool)</signal>
   <receiver>auto_name_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>common_name</sender>
   <signal>toggled(bool)</signal>
   <receiver>common_name_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_name</sender>
   <signal>toggled(bool)</signal>
   <receiver>type_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_name</sender>
   <signal>toggled(bool)</signal>
   <receiver>qualifier_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_name</sender>
   <signal>toggled(bool)</signal>
   <receiver>size_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_name</sender>
   <signal>toggled(bool)</signal>
   <receiver>scale_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>common_name</sender>
   <signal>toggled(bool)</signal>
   <receiver>qualifier_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sampler_name</sender>
   <signal>toggled(bool)</signal>
   <receiver>type_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>attribute_name</sender>
   <signal>toggled(bool)</signal>
   <receiver>attribute_name_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
